pyne/endl.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python
+
+"""Module for parsing and manipulating data from ENDL evaluations.
+
+For the moment, classes and functions in this module only implement the
+specifications relevant to the EEDL (Evaluated Electron Data Library) and the
+EPDL (Evaluated Photon Data Library). The formats are described by the
+following documents:
+
+"ENDL type formats for the Livermore Evaluated Photon Data Library, EPDL"
+https://www.oecd-nea.org/dbdata/data/manual-endf/nds_eval_epdl.pdf
+
+"ENDL type formats for the Livermore Evaluated Electron Data Library, EEDL"
+https://www.oecd-nea.org/dbdata/data/manual-endf/nds_eval_eedl.pdf
+
+For more information, contact Davide Mancusi <davide.mancusi@cea.fr>.
+"""
+from __future__ import print_function, division, unicode_literals
+
+from warnings import warn
+from pyne.utils import QAWarning
+
+import pyne.rxdata as rx
```

Comment:
I mistook this as a `rxname` because [our style guide says that rx stands for "reaction"](http://pyne.io/devsguide/style_guide.html#variable-name-conventions).  Would you mind just doing `from pyne import rxname`? Thanks!